class.p8

```diff
@@ -23,16 +21,3 @@ function class(parent, proto)
  proto.__index = proto
  return proto
 end
-
--- -- usage example:
--- thingclass = class(nil, {x=0, y=0, __init = function(self, x, y) self.x=x self.y=y})
--- movablethingclass = class(thingclass, {move = function() x=x+1 y=y+1 end})
--- amt = movablethingclass(3,5)
--- -- this calls getmetatable(amt).__init(amt,3,5)
--- -- but getmetatable(amt) doesn't have __init, so getmetatable(getmetatable(amt)).__index.__init is called
--- -- this resolves to thingclass.__init which does exist
--- -- so in the end this does thingclass.__init(amt,3,5)
--- amt.move()
--- print(amt.x)
--- print(amt.y)
--- -- amt.x is now 4, amt.y is now 6
```

Comment:
Did you intend to remove this without replacing it elsewhere in comments/docs? It could go in `@usage` for the class constructor.

Reply:
Yes, something about it screws up ldoc. At any rate, stuff like this can go into dedicated examples files that don't get docs generated for them. We should strive to keep the code just code and docs as usage examples are unlikely to get updated over time.

Reply:
Created #10 to follow up on this.